docs/guides/insar_product_guide.md

```diff
@@ -58,17 +58,17 @@ In contrast to the (physical) baseline, the *temporal baseline* refers to the ti
 
 *Table 1: Temporal baselines and what they measure. Different geophysical phenomena can be detected based upon the temporal baseline. In general, the longer the temporal baseline, the smaller the motion that can be detected.*
 
-## InSAR Workflow
-
-The InSAR workflow used in HyP3 was developed by ASF using GAMMA software.  The steps include pre-processing steps, interferogram preparation, and product creation.  Once these steps are performed, an output product package will be created.  See [product packaging](#product-packaging) for details on the individual files in the package.  
-
-### Pre-Processing
+## Ordering On Demand InSAR Products
+[On Demand InSAR](https://storymaps.arcgis.com/stories/68a8a3253900411185ae9eb6bb5283d3) products are generated using ASF's HyP3 platform. Jobs can be submitted for processing using the [Vertex](https://search.asf.alaska.edu/#/) data portal, the [HyP3 Python SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/) or the [HyP3 API](https://hyp3-docs.asf.alaska.edu/using/api/). 
 
-Pre-processing steps prepare the SAR images to be used in interferometry.  The pre-processing steps include image selection, ingest (including calibration), creation of a suitable DEM, and calculation of the burst overlap.
+### Vertex 
+InSAR pairs are selected in [Vertex](https://search.asf.alaska.edu/#/) using the [SBAS Tool](https://docs.asf.alaska.edu/vertex/sbas/). The tool is designed for generating time series of InSAR pairs with short baselines for SBAS analysis, but it is currently the only interface where InSAR pairs can be selected for processing. Even if you are only interested in very specific pairings, you will need to submit them from the SBAS search interface. Refer to our [step-by-step tutorial](https://storymaps.arcgis.com/stories/68a8a3253900411185ae9eb6bb5283d3) for guidance on using the SBAS tool for this process.
 
-#### Finding an InSAR Pair
+### HyP3 SDK and API
+You may still find the Geographic Search and SBAS tool functionality useful when submitting InSAR jobs via the [SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/) or [API](https://hyp3-docs.asf.alaska.edu/using/api/). When you generate a list of pairs in the SBAS Tool, there is an option at the top of the results list to download the list as a CSV file. This list can then be used to populate your SDK or API request.
```

Comment:
@asjohnston-asf @jhkennedy Any hot tips for how you usually assemble image pairs for submission via the SDK or API? I *assume* that the CSV export from SBAS is handy, but I've never gone through that workflow.

Reply:
We have a handy example notebook that does just that:
https://nbviewer.jupyter.org/github/ASFHyP3/hyp3-sdk/blob/main/docs/sdk_example.ipynb#Submitting-Sentinel-1-InSAR-jobs

Reply:
ooh, you're talking about copying and using a list from the sbas tool; I'm not sure why you'd do that if SBAS tool gave you pairs you cared about, but for geographic it'd look like our example

Reply:
we don't talk about how to export the list or get it into python though

Reply:
Thanks! I'll link that example notebook. 
I suppose it's true that if you've already used SBAS to generate a good list, you may as well just submit it from Vertex. ;) 
If you can't think of any use cases for downloading the CSV, I'll just remove that reference from this context.